third_party/move/tools/move-package/tests/test_runner.rs

```diff
@@ -28,21 +33,16 @@ use tempfile::tempdir;
 const COMPILE_EXT: &str = "compile";
 const MODEL_EXT: &str = "model";
 
-pub fn run_test(path: &Path) -> datatest_stable::Result<()> {
-    package_hooks::register_package_hooks(Box::new(TestHooks()));
-    let update_baseline = read_env_update_baseline();
-    if path
-        .components()
-        .any(|component| component == Component::Normal(OsStr::new("deps_only")))
-    {
-        return Ok(());
+fn run_test_impl(path: &Path, v2_flag: bool) -> datatest_stable::Result<String> {
+    let mut compiler_config = CompilerConfig {
+        known_attributes: KnownAttribute::get_all_attribute_names().clone(),
```

Comment:
Add known_attributes to the compiler config, which leads to update of exp files in this PR